src/core/card/Card.tsx

```diff
@@ -0,0 +1,49 @@
+import { IconArrowRight } from 'hds-react'
+import React from 'react'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import classNames from 'classnames'
+
+import styles from './card.module.scss'
+
+export type CardProps = {
+  imageUrl?: string
+  subTitle?: string
+  text?: string
+  title?: string
+  url?: string
+  className?: string
+}
+
+export default function Card({
+  imageUrl,
+  subTitle,
+  text,
+  title,
+  url,
+  className,
+}: CardProps) {
+  return (
+    <div className={classNames(styles.cardWrapper, className)}>
+      {imageUrl && (
+        <div
+          className={styles.imageWrapper}
+          style={{
+            backgroundImage: `url(${imageUrl})`,
+          }}
+        />
+      )}
```

Comment:
Actually, one more thing! The SidebarContentCard uses Img-tag to show the card image. Should the Card do so also? 
```suggestion
      {imageUrl && (
        <div className={styles.image}>
          <Img src={imageUrl} alt={imageAlt ?? ""} />
        </div>
      )}
```

Reply:
Also, should the card have some kind of default image when there is none given? Here is an example from my collection that I'm doing 
<img width="712" alt="image" src="https://user-images.githubusercontent.com/389204/168008539-c19dc29a-ca81-4296-b354-d4ca9ae3602a.png">